ThirdPartyAdapters/vungle/vungle/src/main/java/com/vungle/mediation/VungleManager.java

```diff
@@ -159,12 +149,11 @@ public void removeActiveBanner(String placementId, String adapterId) {
      *
      * @param placementId
      */
-    public void cleanUpBanner(String placementId) {
+    void cleanUpBanner(String placementId) {
         Log.d(TAG, "cleanUpBanner");
         Pair<String, VungleNativeAd> pair = activeBannerAds.get(placementId);
         if (pair != null) {
-            String adapterId = pair.first; //TODO why not all elements, if somehow we end up with more than one
```

Comment:
This is one of the main reason why we have adapterId implementation in first place , as due to bugs in AdMobs adapter code we could end up with scenario where we have multiple instances / requests, and in that scenario calling cleanUpBanner with only Placement could be problematic
If I remember we could have multiple requestBannerAd , and also at time we wont get onDestroy, or any callback to dismiss previous banner, and get request for new instance for same Placement.